apps/docs-e2e/src/docs.spec.ts

```diff
@@ -10,35 +12,178 @@ const test = base.extend<{ docsLayoutPage: DocsLayout }>({
   },
 })
 
-const DOCS_SITE_URL = 'http://localhost:3000/docs'
-const CUHACKING_2025_PLATFORM_GITHUB_REPOSITORY_URL = 'https://github.com/cuhacking/2025'
-const CUHACKING_2025_PLATFORM_GITHUB_PROJECT_BOARD_URL = 'https://github.com/orgs/cuhacking/projects/4'
+const GITHUB_BASE_URL = 'https://github.com/cuhacking'
+const GITHUB_ORG_BASE_URL = 'https://github.com/orgs/cuhacking'
+const DOCS_BASE_URL = 'http://localhost:3000'
 
-test('should contain page title', async ({ docsLayoutPage }) => {
-  await expect(docsLayoutPage.page).toHaveTitle(/Welcome to the Docs/)
-})
+const CUHACKING_2025_PLATFORM_GITHUB_PROJECT_BOARD_URL = `${GITHUB_ORG_BASE_URL}/projects/4`
+const CUHACKING_2025_PLATFORM_GITHUB_REPOSITORY_URL = `${GITHUB_BASE_URL}/2025`
+const CUHACKING_2025_PLATFORM_GITHUB_INDEX_PAGE_URL = `${GITHUB_BASE_URL}/2025/blob/main/apps/docs/src/content/docs/index.mdx`
+const CUHACKING_2025_LANDING_PAGE_GITHUB_REPOSITORY_URL = `${GITHUB_BASE_URL}/landing-page`
+
+const CUHACKING_2025_DOCS_URL = `${DOCS_BASE_URL}/docs`
+
+const CUHACKING_2025_LANDING_PAGE_URL = 'https://www.cuhacking.ca/'
 
-test.describe('should contain desktop header elements', {
+/* ---------------- MOBILE + DESKTOP + TABLET ---------------- */
+test.describe(`Common MOBILE, TABLET and DESKTOP Layout Elements`, {
```

Comment:
wait sorry, changing my review, won't these fail the pipeline?

Reply:
the pipeline would run all the "projects" and the desktop would fail on these everytime?